index.html

```diff
@@ -220,16 +221,16 @@ <h2>Target Audience</h2>
 		<p>Each conformance requirement indicates the audience to which it applies.</p>
 		<p>Although this specification is applicable to the above audiences, it is not specifically targeted to, nor is it intended to be the sole source of information for, any of these audiences. The following documents provide important supporting information:</p>
 		<ul>
-			<li><cite><a href="https://www.w3.org/TR/wai-aria-practices-1.1/"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices 1.1</a></cite> addresses authoring recommendations for HTML, and is also of interest to developers of authoring tools and conformance checkers.</li>
-			<li><cite><a href="https://www.w3.org/TR/core-aam-1.1/">Core Accessibility API Mappings 1.1</a></cite> addresses developers of <a class="termref">user agents</a> and <a class="termref">assistive technologies</a>.</li>
-			<li><cite><a href="https://www.w3.org/TR/accname-aam-1.1/">Accessible Name and Description: Computation and API Mappings 1.1</a></cite> also addresses developers of <a class="termref">user agents</a> and <a class="termref">assistive technologies</a>.</li>
+			<li><cite><a href="" class="practices"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</a></cite> addresses authoring recommendations for HTML, and is also of interest to developers of authoring tools and conformance checkers.</li>
+			<li><cite><a href="" class="core-mapping">Core Accessibility API Mappings</a></cite> addresses developers of <a class="termref">user agents</a> and <a class="termref">assistive technologies</a>.</li>
+			<li><cite><a href="" class="accname">Accessible Name and Description: Computation and API Mappings</a></cite> also addresses developers of <a class="termref">user agents</a> and <a class="termref">assistive technologies</a>.</li>
```

Comment:
These 3 in particular should probably keep their hardcoded version number? And perhaps they need a versioned reference as well? These are the only 3 that I originally thought were "suite" references.
(Also, need to update accname's title).

```suggestion
			<li><cite><a href="https://www.w3.org/TR/wai-aria-practices-1.2/"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices 1.2</a></cite> addresses authoring recommendations for HTML, and is also of interest to developers of authoring tools and conformance checkers.</li>
			<li><cite><a href="https://www.w3.org/TR/core-aam-1.2/">Core Accessibility API Mappings 1.2</a></cite> addresses developers of <a class="termref">user agents</a> and <a class="termref">assistive technologies</a>.</li>
			<li><cite><a href="https://www.w3.org/TR/accname-aam-1.2/">Accessible Name and Description Computation 1.2</a></cite> also addresses developers of <a class="termref">user agents</a> and <a class="termref">assistive technologies</a>.</li>
```

Reply:
lets just change these to the triple bracket version